src/Storage/Storage.Management/help/New-AzStorageContext.md

```diff
@@ -163,14 +187,59 @@ PS C:\> $Context = New-AzStorageContext -StorageAccountName "myaccountname" -Use
 
 This command creates a context by using the OAuth (Azure AD) Authentication.
 
+### Example 11: Create a context by specifying a storage account name, storage account key and custom blob endpoint
+```
+PS C:\> New-AzStorageContext -StorageAccountName "MyAccountName" -StorageAccountKey "< Storage Key for MyAccountName ends with == >" -BlobEndpoint "https://MyAccountName.blob.core.windows.net/"
```

Comment:
please use low case account name like "myaccountname", same in the Uri.
Same in all following new samples.